### PR TITLE
Misc: update dependencies

### LINKED ISSUE #1347: Update dependencies?
`estraverse		^1.9.3	4.1.0` and some others (commander, lodash.assign, browserify, jshint).

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/13370481-update-dependencies?utm_campaign=plugin&utm_content=tracker%2F281640&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F281640&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by hzoo**
Fixes #1347 

No update to esprima or estraverse (broken tests due to `ExportDeclaration`) - will have to update both for the new ESTree changes?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -78,11 +78,11 @@
     "xmlbuilder": "^2.6.1"
   },
   "devDependencies": {
-    "browserify": "^9.0.3",
+    "browserify": "^10.1.3",
```
Comment (mrjoelkemp) [inline package.json]: We don't have tests for the browser version. Anything worrisome in that major version bump?

Comment (hzoo) [inline package.json]: Related to `process`:

> The ‘process’ dependency was updated to ~0.11.0, this module is inserted into bundles as the ‘process’ global/dependency. Previously, an unhandled error thrown in a ‘process.nextTick’ task would prevent any subsequent tasks from running, forever. The task queue now recovers from this condition, but may do so on a future browser tick. As part of this update, ‘process.nextTick’ now accepts variadic arguments, passed to the task, added to io.js in 1.8.1.

Or just update to `9.0.8`.

Comment (mrjoelkemp) [inline package.json]: @gero3 any way for you to test that this PR (after generating the bundle) works with your mr doob app?

**Comment by hzoo** [comment]
Ok only updating minor versions.

**Comment by hzoo** [comment]
Not sure if one of the tests isn't calling done correctly or just an issue with travis.

**Comment by zxqfox** [comment]
@hzoo It's 2sec timeout on some test while running instucted code. We need to fix it ;-(

**Comment by mrjoelkemp** [comment]
I restarted the build and it's passing. Couldn't reproduce the timeout locally either.

**Comment by mrjoelkemp** [comment]
LGTM

**Pull request closed without merging**